tests/unittests/TestData/CPACS_30_D150_ducts.xml

```diff
@@ -0,0 +1,6521 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should delete this file. 
1. It is not used in any tests
2. It isn't schema-conform anymore.